pkg/virtctl/expose/expose.go

```diff
@@ -69,6 +70,7 @@ virtualmachineinstance (vmi), virtualmachine (vm), virtualmachineinstancereplica
 	cmd.Flags().StringVar(&strTargetPort, "target-port", "", "Name or number for the port on the VM that the service should direct traffic to. Optional.")
 	cmd.Flags().StringVar(&strServiceType, "type", "ClusterIP", "Type for this service: ClusterIP, NodePort, or LoadBalancer.")
 	cmd.Flags().StringVar(&portName, "port-name", "", "Name of the port. Optional.")
+	cmd.Flags().StringVar(&strIPFamily, "ip-family", "IPv4", "IP family over which the service will be exposed. Valid values are 'IPv4' or 'IPv6'.")
```

Comment:
I would remove the "IPv" prefix, I think `--ip-family=4` `--ip-family=6` is enough.

Reply:
Since the attribute is configured w/ `--ip-family` I'd prefer to stick w/ `IPv4` /  `IPv6`, since it matches the possible values of the k8s `IPFamily` attribute.

https://github.com/kubevirt/kubevirt/blob/d072d7ffcbba8d862e79db7749fd0effe2ca2f4e/vendor/k8s.io/api/core/v1/types.go#L3824

Reply:
I have to admit Miguel convinced me. We have to be aligned with k8s services.